packages/ui/components/form-core/src/validate/ValidateMixin.js

```diff
@@ -460,7 +459,9 @@ export const ValidateMixinImplementation = superclass =>
 
       if (isEmpty) {
         const hasSingleValue = !(/** @type {*  & ValidateHost} */ (this)._isFormOrFieldset);
-        const requiredValidator = this._allValidators.find(v => v instanceof Required);
+        const requiredValidator = this._allValidators.find(
+          v => /** @type {typeof Validator} */ (v.constructor)?.validatorName === 'Required',
```

Comment:
Will `typeof Validator` work since the import for `Validator` was removed?

Reply:
Add back as `typedef`